sdk/monitor/monitor-opentelemetry/src/utils/common.ts

```diff
@@ -68,10 +68,10 @@ export const isAks = (): boolean => {
  * non-Web and non-Function APP: "u" (unknown)
  */
 export const getResourceProvider = (): string => {
-  if (isFunctionApp()) {
+  if (isAppService()) {
```

Comment:
This change still doesn't make sense to me, how this is different to what we have before?, you were checking for FUNCTIONS_WORKER_RUNTIME first, so only true for functions, then if not true you check for WEBSITE_SITE_NAME then is appService, am I missing something here?